### PR TITLE
Move the IngestionService to it's own package (as it's not "meta")

### PR DESCRIPTION
**Pull request by andreidan**

### CODE REVIEW

File: sql/src/main/java/io/crate/metadata/MetaDataModule.java [hunk 1]
```diff
@@ -22,7 +22,7 @@
 package io.crate.metadata;
 
 import io.crate.metadata.cluster.DDLClusterStateService;
-import io.crate.metadata.rule.ingest.IngestionService;
+import io.crate.ingestion.IngestionService;
```
Comment (seut) [inline sql/src/main/java/io/crate/metadata/MetaDataModule.java]: unused import?

File: sql/src/main/java/io/crate/metadata/rule/ingest/IngestRulesMetaData.java [hunk 2]
```diff
@@ -67,7 +67,7 @@ public static IngestRulesMetaData copyOf(@Nullable IngestRulesMetaData oldMetaDa
      * is changed to the given target ident.
      */
     @Nullable
-    static IngestRulesMetaData maybeCopyAndReplaceTargetTableIdents(IngestRulesMetaData oldMetaData,
+    public static IngestRulesMetaData maybeCopyAndReplaceTargetTableIdents(IngestRulesMetaData oldMetaData,
                                                                     String sourceIdent,
                                                                     String targetIdent) {
```
Comment (seut) [inline sql/src/main/java/io/crate/metadata/rule/ingest/IngestRulesMetaData.java]: pls fix formatting

File: sql/src/test/java/io/crate/metadata/rule/ingest/IngestionServiceTest.java [hunk 3]
```diff
@@ -22,6 +22,7 @@
 
 package io.crate.metadata.rule.ingest;
 
+import io.crate.ingestion.IngestionService;
```
Comment (seut) [inline sql/src/test/java/io/crate/metadata/rule/ingest/IngestionServiceTest.java]: pls move test class as well

Comment (andreidan) [inline sql/src/test/java/io/crate/metadata/rule/ingest/IngestionServiceTest.java]: ah, yes

**Pull request merged**